ee_plugin/ee_plugin.py

```diff
@@ -209,10 +209,17 @@ def unload(self):
         if self.menu:
             self.iface.pluginMenu().removeAction(self.menu.menuAction())
 
-        if self.toolButton:
-            self.toolButton.deleteLater()
+        try:
+            if self.toolButton:
+                self.toolButton.deleteLater()
+        except RuntimeError as e:
+            print(f"Error deleting toolButton: {e}")
```

Comment:
Is our logger not available here?

Reply:
yes, I think it's not initialized from previous experimentation.